css-four-column-layout/styles.css

```diff
@@ -0,0 +1,78 @@
+.container {
+  width: 80%;
+  margin: auto;
+  background-color: #f7f8fa;
+  padding-left: 8rem;
+  padding-right: 8rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.header-row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 20pt;
+  font-family: 'Barlow', sans-serif;
+  font-weight: bold;
+}
+
+.four-phone-pictures-div {
+  display: flex;
+}
+
+.column {
+  flex-wrap: wrap;
+}
+
+.column > img.phone-pic {
+  width: 85%;
+  margin: 0.5rem;
+  display: block;
+  position: relative;
+  left: 1rem;
+}
+
+.column > img.icon {
+  width: 20%;
+}
+
+.footer-column > h3 {
+  font-family: 'Barlow', sans-serif;
+  font-weight: 700;
+}
+
+.footer-column > p {
+  padding-top: 1rem;
+}
+
+.footer-row {
+  display: flex;
+  justify-content: space-around;
+  text-align: center;
+  font-family: 'Barlow', sans-serif;
+}
+
+.one {
+  position: relative;
+  right: 0.1rem;
+}
+
+.two {
+  position: relative;
+  right: 0.1rem;
+}
+
+.three {
+  position: relative;
+  left: 0.1rem;
+}
+
+.four {
+  position: relative;
+  left: 0.1rem;
+}
```

Comment:
You should just make this into two reusable classes, the declaration block is duplicated once for each of these